src/mongo/s/mongos_main.cpp

```diff
@@ -67,7 +67,10 @@
 #include "mongo/db/auth/authorization_manager.h"
 #include "mongo/db/auth/authz_manager_external_state.h"
 #include "mongo/db/auth/authz_manager_external_state_s.h"
+#ifdef PERCONA_OIDC_ENABLED
+#include "mongo/db/auth/oidc/oidc_server_parameters_logger.h"
 #include "mongo/db/auth/oidc/oidc_identity_providers_registry.h"
+#endif
```

Comment:
Whitespace style of this block differs from the style of similar block in mongod_main.cpp. Please unify.

Reply:
Sorry, still different styles (last version seems to swap styles).

Reply:
Fixed.